scene/2d/audio_stream_player_2d.cpp

```diff
@@ -430,6 +461,39 @@ void AudioStreamPlayer2D::_on_bus_renamed(int p_bus_index, const StringName &p_o
 	notify_property_list_changed();
 }
 
+bool AudioStreamPlayer2D::_set(const StringName &p_name, const Variant &p_value) {
+	if (playback_parameters.has(p_name)) {
+		ParameterData &pd = playback_parameters[p_name];
+		pd.value = p_value;
+		for (int i = 0; i < stream_playbacks.size(); i++) {
+			stream_playbacks[i]->set_parameter(pd.path, pd.value);
+		}
+		return true;
+	}
+
+	return false;
+}
+
+bool AudioStreamPlayer2D::_get(const StringName &p_name, Variant &r_ret) const {
+	if (playback_parameters.has(p_name)) {
+		r_ret = playback_parameters[p_name].value;
+		return true;
+	}
+	return false;
+}
+
+void AudioStreamPlayer2D::_get_property_list(List<PropertyInfo> *p_list) const {
+	if (stream.is_valid()) {
```

Comment:
This can technically also be an early return, though not sure if it fits this method.